index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```

Comment:
This looks great. Maybe add + <h2 hidden>Game over, click refresh to play again!</h2>